$NetBSD: patch-configurein,v 1.2 2013/01/26 22:20:25 wiz Exp $

First chunk: automake-1.13 compat.
Rest: Added the necessary quoting for m4 macros.

--- configure.in.orig	2001-07-15 21:25:20.000000000 +0000
+++ configure.in
@@ -24,7 +24,7 @@ dnl ====================================
 SPACERACER_VERSION=0.2.3
 SDL_VERSION=1.1.0
 AM_INIT_AUTOMAKE(spaceracer,$SPACERACER_VERSION)
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 
 
 dnl ==========================================
@@ -103,15 +103,6 @@ CXXFLAGS="$CFLAGS"
 
 
 
-dnl ==========================================
-dnl  Doit-t\'on afficher les info de debug ?
-dnl ==========================================
-AC_DIVERT_PUSH(AC_DIVERSION_NOTICE)dnl
-  debug_default=no
-  profile_default=no
-  profile_debug=no
-AC_DIVERT_POP()dnl
-
 AC_ARG_ENABLE(debug, [  --enable-debug=[no/minimum/yes] turn on debugging [default=$debug_default]],,enable_debug=$debug_default)
 AC_ARG_ENABLE(debug, [  --enable-profile=[no/yes] turn on profiling [default=$profile_default]],,profile_debug=$profile_default)
 
@@ -151,23 +142,6 @@ if eval "test x$enable_spaceracerdir != 
   fi
 fi)
 
-
-
-
-
-
-dnl si c\'est Gcc, on rajoute l\'option -Wall que si il n\'y si trouve pas déjà
-changequote(,)dnl
-if test "x$GCC" = "xyes"; then
-  case " $CXXFLAGS " in
-  *[\ \	]-Wall[\ \	]*) ;;
-  *) CXXFLAGS="$CXXFLAGS -Wall" ;;
-  esac
-fi
-changequote([,])dnl
-
-
-
 dnl ======================================
 dnl  Détection des headers standart
 dnl ======================================
@@ -180,26 +154,26 @@ dnl ====================================
 
 AC_CHECK_LIB(jpeg,
              jpeg_start_decompress,
-	     AC_CHECK_HEADERS(jpeglib.h,
+	     [AC_CHECK_HEADERS(jpeglib.h,
 	                      JPEGLIB="-ljpeg",
-			      AC_MSG_WARN(jpeg header not found)),
+			      AC_MSG_WARN(jpeg header not found))],
 	     ,
 	     -lm)
 
 AC_CHECK_LIB(z,
              gzopen,
-	     AC_CHECK_HEADERS(zlib.h,
+	     [AC_CHECK_HEADERS(zlib.h,
 	                      ZLIB="-lz",
-			      AC_MSG_WARN(zlib header not found)),
+			      AC_MSG_WARN(zlib header not found))],
 	     ,
 	     -lm)
 if test "$ac_cv_header_zlib_h" = "yes"
 then
 AC_CHECK_LIB(png,
              png_get_IHDR,
-	     AC_CHECK_HEADERS(png.h,
+	     [AC_CHECK_HEADERS(png.h,
 	                      PNGLIB="-lpng $ZLIB",
-			      AC_MSG_WARN(png header not found)),
+			      AC_MSG_WARN(png header not found))],
 	     ,
 	     $ZLIB -lm)
 fi
@@ -223,31 +197,31 @@ XLIBS="$X_EXTRA_LIBS -lX11"
 
 AC_CHECK_LIB(Xext,
              XShmQueryExtension,
-	     AC_CHECK_HEADERS(X11/extensions/XShm.h,
+	     [AC_CHECK_HEADERS(X11/extensions/XShm.h,
 	                      XLIBS="-lXext $XLIBS",
-			      AC_MSG_WARN(libXext Not found)),
+			      AC_MSG_WARN(libXext Not found))],
 	     ,
 	     $XLFLAGS $X_PRE_LIBS $XLIBS)
 
 AC_CHECK_LIB(Xmu,
              XmuLookupStandardColormap,
-             AC_CHECK_HEADER(X11/Xmu/WinUtil.h,
+             [AC_CHECK_HEADER(X11/Xmu/WinUtil.h,
                              XLIBS="-lXmu $XLIBS",
-                             AC_MSG_WARN(libXmu Not found)),
-             AC_CHECK_LIB(Xmu,
+                             AC_MSG_WARN(libXmu Not found))],
+             [AC_CHECK_LIB(Xmu,
 	                  XmuUpdateMapHints,
-                          AC_CHECK_HEADER(X11/Xmu/WinUtil.h,
+                          [AC_CHECK_HEADER(X11/Xmu/WinUtil.h,
                                           XLIBS="-lXmu -lXt $XLIBS",
-                                          AC_MSG_WARN(XMU library not found)),
+                                          AC_MSG_WARN(XMU library not found))],
                                           AC_MSG_WARN(XMU library not found),
-                          $XLFLAGS $X_PRE_LIBS -lXt $XLIBS),
+                          $XLFLAGS $X_PRE_LIBS -lXt $XLIBS)],
              $XLFLAGS $X_PRE_LIBS $XLIBS)
 
 AC_CHECK_LIB(Xi,
              XGetExtensionVersion,
-	     AC_CHECK_HEADERS(X11/extensions/XI.h,
+	     [AC_CHECK_HEADERS(X11/extensions/XI.h,
 	                      XLIBS="-lXi $XLIBS",
-			      AC_MSG_WARN(libXi Not found)),
+			      AC_MSG_WARN(libXi Not found))],
 	     ,
 	     $XLFLAGS $X_PRE_LIBS $XLIBS)
 
